scripts/install-contributor-dependencies.sh

```diff
@@ -0,0 +1,27 @@
+# Install brew
+if ! command -v brew &> /dev/null
+then
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+fi
```

Comment:
Should this be wrapped in a check for `uname == "Darwin"`?

Reply:
Similar to below, maybe we should just rename this file as macos specific script and note that from the contributing guide